xdggs/plotting.py

```diff
@@ -1,14 +1,28 @@
 import numpy as np
 
 
+def geoarrow2table(polygons):
+    from arro3.core import Array, Schema, Table
+
+    array = Array.from_arrow(polygons)
+    field = array.field.with_name("geometry")
+    schema = Schema([field])
+    return Table.from_arrays([array], schema=schema)
+
+
+def column_from_numpy(arr):
+    from arro3.core import Array, ChunkedArray
+
+    return ChunkedArray([Array.from_numpy(arr)])
```

Comment:
As of arro3 0.4 this should be zero-copy from numpy memory (for C contiguous numeric input)!